testdata/project-v3-config/go.mod

```diff
@@ -1,6 +1,6 @@
 module sigs.k8s.io/kubebuilder/testdata/project-v3-config
 
-go 1.18
+go 1.19
 
 require (
```

Comment:
@camilamacedo86 A side question. Kubebuilder seems to have updated to ginkov2, but the testdata is still `github.com/onsi/ginkgo v1.16.5`. Should this be updated? And if so where?